### PR TITLE
feat:add pix static qrcode section

### PR DESCRIPTION
**Pull request by agabiandrade**
## Description

feat: new section for pix: create, update and detail for static QRCode
fix: create qrcode with due date: the calling method was GET instead of POST.
-

### CODE REVIEW

File: content/pt/docs/referencia-da-api/pix/recebimento/atualizar-qrcode-estatico/_index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  "account_id": "<uuid>",
+  "participant_ispb": "<string>",
+  "key": "<string>",
+  "key_type": "<string>",
+  "transaction_id": "<string>",
+  "amount": "<integer>",
+  "additional_information": "<string>",  
+  "request_id": "<string>",
+  "created_at": "<iso8631>",
+  "updated_at": "<iso8631>",
```
Comment (pedroyremolo) [inline content/pt/docs/referencia-da-api/pix/recebimento/atualizar-qrcode-estatico/_index.md:64]: well, I'm not pretty sure about this. But the ISO number for date-time is 8601, isn't it? also, I believe a date-time prefix would stand clear for the field.

File: content/pt/docs/referencia-da-api/pix/recebimento/criar-qrcode-estatico/_index.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  "account_id": "<uuid>",
+  "participant_ispb": "<string>",
+  "key": "<string>",
+  "key_type": "<string>",
+  "transaction_id": "<string>",
+  "amount": "<integer>",
+  "additional_information": "<string>",  
+  "request_id": "<string>",
+  "created_at": "<iso8631>",
+  "updated_at": "<iso8631>",
```
Comment (pedroyremolo) [inline content/pt/docs/referencia-da-api/pix/recebimento/criar-qrcode-estatico/_index.md:69]: same as above

File: content/pt/docs/referencia-da-api/pix/recebimento/detalhar-qrcode-estatico/_index.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  "account_id": "<uuid>",
+  "participant_ispb": "<string>",
+  "key": "<string>",
+  "key_type": "<string>",
+  "transaction_id": "<string>",
+  "amount": "<integer>",
+  "additional_information": "<string>",  
+  "request_id": "<string>",
+  "created_at": "<iso8631>",
+  "updated_at": "<iso8631>",
```
Comment (pedroyremolo) [inline content/pt/docs/referencia-da-api/pix/recebimento/detalhar-qrcode-estatico/_index.md:53]: same here

**Review by pedroyremolo — DISMISSED** [review]
just some small comments, tell me what do u think

File: content/pt/docs/referencia-da-api/pix/recebimento/detalhar-qrcode-estatico/_index.md (hunk 3 above)
Comment (agabiandrade) [reply content/pt/docs/referencia-da-api/pix/recebimento/detalhar-qrcode-estatico/_index.md:53]: agreed!

File: content/pt/docs/referencia-da-api/pix/recebimento/criar-qrcode-estatico/_index.md (hunk 2 above)
Comment (agabiandrade) [reply content/pt/docs/referencia-da-api/pix/recebimento/criar-qrcode-estatico/_index.md]: agreed!

File: content/pt/docs/referencia-da-api/pix/recebimento/atualizar-qrcode-estatico/_index.md (hunk 1 above)
Comment (agabiandrade) [reply content/pt/docs/referencia-da-api/pix/recebimento/atualizar-qrcode-estatico/_index.md:64]: agreed!

**Review by pedroyremolo — APPROVED** [review]

**Pull request merged**